### PR TITLE
Profpoint trap event collector

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-3170

## Description
Event collector 

## Minimum version of Cortex XSOAR
- [x] 6.8.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/README.md
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/README.md:3]:
```suggestion
## Configure Proofpoint Threat Response Event Collector on Cortex XSIAM
```

File: Packs/ProofpointThreatResponse/ReleaseNotes/1_0_13.md
Comment (dantavori) [inline Packs/ProofpointThreatResponse/ReleaseNotes/1_0_13.md:8]:
```suggestion
Use the Proofpoint Threat Response integration to orchestrate and automate incident response. (Available from Cortex XSIAM 3.3.0).

#### Modeling Rules
##### New: Proofpoint Threat Response Modeling Rule
Added the *Email* modeling rule. (Available from Cortex XSIAM 3.3.0).
```

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector_description.md
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector_description.md:6]:
```suggestion  
```

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector_description.md
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector_description.md:8]:
```suggestion
```

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py:308]:
```suggestion
    demisto.debug(f'Fetched {len(incidents)} events')
```

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py:260]:
```suggestion
        demisto.debug(
```

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py:281]:
```suggestion
    events = []
```

**Review by dantavori — CHANGES_REQUESTED** [review]

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py:14]:
```suggestion
    Client will implement the service API, and should not contain any logic.
```

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml:24]: please move to bottom of the params

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml:67]: please verify this description with a TPM

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector_description.md
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector_description.md:6]:
```suggestion
**Note:** if the **fetch-events** command results in a timeout, consider changing **fetch_delta** and **fetch_limit** parameters to reduce the number of API calls and results.
```

File: Packs/ProofpointThreatResponse/ModelingRules/ProofpointThreatResponseModelingRules/ProofpointThreatResponseModelingRules.yml
Comment (dantavori) [inline Packs/ProofpointThreatResponse/ModelingRules/ProofpointThreatResponseModelingRules/ProofpointThreatResponseModelingRules.yml:9]:
```suggestion
```
as Shahaf said last week, if the issues with the new modeling rules are not resolved until you are ready to merge, let's remove them and we'll add them later

File: Packs/ProofpointThreatResponse/pack_metadata.json
Comment (dantavori) [inline Packs/ProofpointThreatResponse/pack_metadata.json:5]: should be major update for new integration - 2.0.0

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml:1]: shouldn't it be `Analytics & SIEM`?

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py:341]:
```suggestion
    base_url = params.get('url')
```

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py
Comment (dantavori) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.py:317]: please add a docstring

**Review by dantavori — COMMENTED** [review]
looks great, please assign a doc reviewer
I will probably approve after the demo next week

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml
Comment (daryakoval) [reply Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml:1]: No, it is the same as Integration.

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml
Comment (daryakoval) [reply Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml:67]: Meital is out of the office.  This is the description of the integration that fetches incidents.

**Comment by ShirleyDenkberg** [comment]
@daryakoval Shouldn't there be a pack README?
@dantavori Doc review completed.

**Review by dantavori — APPROVED** [review]

**Comment by daryakoval** [comment]
Force merge needed: Proofpoint pack was only for "xsoar" marketplace. I added event collector and added to the pack metadata "marketplacev2" too. 
The build fails on the "pre-update" step, since before the update there is no such pack in the marketplacev2, and after the update the "post-update" passes.

**Pull request merged**
